lms/djangoapps/instructor/views/api.py

```diff
@@ -1644,12 +1644,10 @@ def calculate_grades_csv(request, course_id):
     try:
         instructor_task.api.submit_calculate_grades_csv(request, course_key)
         success_status = _("Your grade report is being generated! You can view the status of the generation task in the 'Pending Instructor Tasks' section.")
-        return JsonResponse({"status": success_status})
+        return JsonResponse({"status": success_status}, status=202)
```

Comment:
It doesn't actually return two statuses; there's just an unfortunate naming collision in the JsonResponse class.  The status field within the dictionary corresponds to the status message which is displayed to the user upon acceptance of their request for report generation.  The status parameter is the HTTP response code which is returned to their browser.